container.go

```diff
@@ -351,6 +301,38 @@ func createContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
 		c.mounts = mounts
 	}
 
+	return c, nil
+}
+
+func createContainers(pod *Pod, contConfigs []ContainerConfig) ([]*Container, error) {
```

Comment:
Bad choice of words. The question boils down to: "why doesn't `createContainers()` call `createContainer()`?